### DISCUSSION TITLE
No response from Github Support for over 2 weeks, I really need my account back ┭┮﹏┭┮

### DISCUSSION THREAD

**Discussion by bathysaurus** [community/community#62338] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My account has also been flagged for no reason, I have not engaged in any behavior that violates community guidelines, but I cannot even log in now.

My Github account is essential to me, but it has been over 2 weeks since I sent the request form to the Github Support Team, and I have followed up on my initial ticket and replied to the initial email, but still no response.

Please help me since it's so significant to me, thank you all.

Ticket ID: 2263295

**Comment by JiveyGuy** [comment]
Wish you luck in this! 

---- Free Hk ♥ ----

**Comment by DSDmark** [comment]
Hi @bathysaurus,

**GitHub support** teams can be slow to respond, especially to more complex issues. Keep following up on your initial ticket. You can try to connect directly with GitHub mail **support@github.com**.

Explain exactly what happened, what actions you've already taken to resolve the issue yourself, and why access to your account is important. The more information you can provide, the better they can assist you.

And I have nothing to write on that matter except wait for response of GitHub.

**Comment by chrisstef** [comment]
Dear GitHub Support Team,

I am reaching out regarding an issue with my GitHub account that I believe requires your attention.

Upon attempting to access my account from incognito mode or any other page where I am not authenticated with GitHub, I consistently encounter a 404 "page not found" error. Additionally, I have received a message stating that my account has been flagged. This has greatly hindered my ability to effectively collaborate with my team, utilize GitHub Copilot, and engage in the application process for various positions.

Given the importance of GitHub in my daily workflow, particularly in light of job applications and team collaboration, I kindly request a thorough investigation into the status of my account. Restoring full access to my account would greatly facilitate my productivity and enable me to continue contributing to projects.

Ticket: #2610547
username: chrisstef

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Comment by ghost** [comment]
Don't want to be a nuisance but I've also been waiting for over 3 weeks now to have my account reinstated so I can start putting a link to it in my resume and apply actively for positions.

Help @github-staff!

**Comment by AimanMaguzPaycasso** [comment]
We had a ticket that has been now been opened since November  with the urgent title Ticket number:  #3105025) we still haven't received a response.
There was direct email sent to the support team [support@github.com] and even reopened an old case just hoping for anything from them but very little joy.
Now we are truly stuck with a GitHub account that we can no longer use and tasks we can no longer function.

Please Help @github-staff!
Regards

**Reply by shinybrightstar** [reply]
Hi @AimanMaguzPaycasso 
This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).


This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.


As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
